plugins/catalog/src/components/CatalogTable/columns.tsx

```diff
@@ -184,12 +184,23 @@ export const columnFactories = Object.freeze({
     key: string,
     options?: { title?: string; defaultValue?: string },
   ): TableColumn<CatalogTableRow> {
+    function formatContent(keyLabel: string, entity: Entity): string {
+      const labels: Record<string, string> | undefined =
+        entity.metadata?.labels;
+      return (labels && labels[keyLabel]) || '';
+    }
+
     return {
       title: options?.title || 'Label',
       field: 'entity.metadata.labels',
```

Comment:
I'm not sure whether it works but about this instead?

```suggestion
      field: `entity.metadata.labels.${key}`,
```

Reply:
> I'm not sure whether it works but what about this instead?

I tested it and it doesn't work the way you mentioned.